src/BloomExe/Collection/CollectionSettings.cs

```diff
@@ -45,36 +45,40 @@ public class CollectionSettings
 				// we then switched to having code keep the page number in data-page-number,
 				// so we can't make use of that CSS feature anymore but want to keep the same
 				// list and keep working for users of previous versions.
-				// In this dictionary, we're pairing css counting styles (the key) with either
-				// a Microsoft culture that happens to implement it or, if that can't be found,
-				// then the 10 digits of used by the script. As a side benefit, this will allow us to support
+				// In this dictionary, we're pairing css counting styles (the key) with
+				// the 10 digits used by the script. As a side benefit, this will allow us to support
 				// other number systems, if people request them (so long as they can be represented by just
 				// replacing digits).
-				{ "Arabic-Indic", "ar-SA"}, // not certain that this is correct one
-				{ "Armenian", "hy-AM"},
-				{ "Upper-Armenian", "hy-AM"},
+				// In many cases, as commented, the ten digits were obtained from Microsoft cultures using this expression:
+				// new CultureInfo(cultureCode).NumberFormat.NativeDigits
+				// Some of the results are empty strings when it doesn't seem they should be (Hebrew, Armenian, Georgian)
+				// This reflects that these systems can't be done with simple digit substitution, so we fall
+				// back to not converting
```

Comment:
> so we fall back to not converting

I would have expected that we would not list these choices; what do we get by listing them but not actually acting on the user's choice?